cognite/neat/rules/models/dms/_schema.py

```diff
@@ -146,8 +147,20 @@ def from_data_model(
         space_write = space_read.as_write()
 
         view_loader = ViewLoader(client)
-        # We need to include parent views in the schema to make sure that the schema is valid.
+
         existing_view_ids = set(views.as_ids())
+
+        # We need to include all views the edges/direct relations are pointing to have a complete schema.
```

Comment:
This is occurs if data model is not made by neat?

Reply:
Yes, it could. In this case, it was actually created by a buggy version of Neat.